integration_tests/src/main/python/hash_aggregate_test.py

```diff
@@ -615,10 +661,9 @@ def test_hash_multiple_mode_query_avg_distincts(data_gen, conf):
 @approximate_float
 @ignore_order
 @incompat
-@pytest.mark.parametrize('data_gen', _init_list_no_nans, ids=idfn)
+@pytest.mark.parametrize('data_gen', _init_list_no_nans[1:2], ids=idfn)
```

Comment:
Using a slice for the data gen is really confusing here. At a minimum we need a comment explaining why we are slicing it. Preferably it is a separate value along with the explanation.

Reply:
Sorry, its my mistake. I added the slice to facilitate debugging, and I forgot to remove it before submission.